atomic_defi_design/Dex/Constants/General.qml

```diff
@@ -357,6 +357,7 @@ QtObject {
         if(type === "ERC-20") return "ETH"
         else if(type === "QRC-20") return "QTUM"
         else if(type === "Smart Chain") return "KMD"
```

Comment:
shouldn't this check work?
it seems doing the same and initial wording matching this check

Reply:
https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/smartdex/src/core/atomicdex/config/coins.cfg.cpp#L102 
vs 
https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/src/core/atomicdex/config/coins.cfg.cpp#L102

Problem is that in `smartdex` branch, there is no match for `Smart chains` in coins.cfg.cpp

A release or two ago it was requested to tag `Smart Chains` as `Komodo Chains` in Smartdex.

In build from https://github.com/KomodoPlatform/atomicDEX-Desktop/actions/runs/1647247078 it was a mix of `Smart chain` and `Komodo chain` and I was seeing errors like this:

![image](https://user-images.githubusercontent.com/35845239/147966160-b654eb9c-316d-49b2-a897-8a1a35a34dda.png)

Reply:
> https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/smartdex/src/core/atomicdex/config/coins.cfg.cpp#L102 vs https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/src/core/atomicdex/config/coins.cfg.cpp#L102
> 
> Problem is that in `smartdex` branch, there is no match for `Smart chains` in coins.cfg.cpp
> 
> A release or two ago it was requested to tag `Smart Chains` as `Komodo Chains` in Smartdex.
> 
> In build from https://github.com/KomodoPlatform/atomicDEX-Desktop/actions/runs/1647247078 it was a mix of `Smart chain` and `Komodo chain` and I was seeing errors like this:
> 
> ![image](https://user-images.githubusercontent.com/35845239/147966160-b654eb9c-316d-49b2-a897-8a1a35a34dda.png)

What about:

```
else if (cfg.type == "Komodo Chain" || cfg.type == "Smart Chain")
        {
            cfg.type = "Smart Chain";
            cfg.coin_type = CoinType::SmartChain;
        }
```

in <https://github.com/KomodoPlatform/atomicDEX-Desktop/blob/dev/src/core/atomicdex/config/coins.cfg.cpp#L102>

this way any other change will not be necessary

Reply:
I'll do this and see if we can get it into a state where the only change needed is in coins json file (in future we should be able to do this in the background perhaps?)

Reply:
> I'll do this and see if we can get it into a state where the only change needed is in coins json file (in future we should be able to do this in the background perhaps?)

Yes we could do it 100% in background but there's is probably an extra tweak needed in frontend to display Komodo Chain instead of Smart Chain